Update n1 waf docs and changelog to reference default log profiles an… (#2110)
nginx/documentation: content/nginx-one-console/waf-integration/log-profiles/configure-log-profiles.md

From NGINX One Console, select **WAF** > **Log Profiles**. In the screen that appears, select **Add Log Profile**. This action opens a screen where you can:

> - In **Start From**, choose a F5 WAF for NGINX [default logging profile]({{< ref "/waf/logging/log-overview.md#default-logging-profile-bundles" >}}) to use as a starting point

Reviewer: ```suggestion
- In **Start From**, select an F5 WAF for NGINX [default logging profile]({{< ref "/waf/logging/log-overview.md#default-logging-profile-bundles" >}}) as a starting point.
```
